docs/src/Modal.doc.js

```diff
@@ -23,85 +32,497 @@ card(
       },
       {
         name: 'footer',
         type: 'React.Node',
-        href: 'sizesExample',
+        description:
+          "Supply the container element(s) or render prop that will be used as Modal's custom footer. See the [Best Practices](#Best-practices) for more info.",
       },
       {
         name: 'heading',
         type: `string | React.Node`,
-        required: false,
-        href: 'heading',
+        description:
+          "The text used for Modal's heading. Be sure to localize this text. See the [Heading variant](#Heading) for more info.",
       },
       {
         name: 'onDismiss',
         type: '() => void',
         required: true,
-        href: 'sizesExample',
+        description:
+          'Callback fired when Modal is dismissed by clicking on the a dismiss-type Footer button or clicking on the backdrop outside of the Modal (if `closeOnOutsideClick` is true).',
       },
       {
         name: 'ref',
         type: "React.Ref<'div'>",
         description: 'Forward the ref to the underlying div element',
-        href: 'refExample',
       },
       {
         name: 'role',
         type: `"alertdialog" | "dialog"`,
         defaultValue: 'dialog',
-        href: 'role',
+        description:
+          'The underlying ARIA role for this Modal. See the [Accessibility Role section](#Role) for more info. ',
       },
       {
         name: 'size',
         type: `"sm" | "md" | "lg" | number`,
         defaultValue: 'sm',
-        description: `sm: 540px, md: 720px, lg: 900px`,
-        href: 'sizesExample',
+        description: `Determines the width of Modal. See the [size variant](#Sizes) for more info.`,
       },
       {
         name: 'subHeading',
         type: `string`,
         required: false,
-        description: `Only renders with \`heading\` strings`,
-        href: 'headingOptions',
+        description: `Subtext for Modal, only renders with \`heading\` strings. See the [sub-heading variant](#Sub-heading) for more info.`,
       },
     ]}
   />,
 );
 
 card(
-  <Example
-    id="sizesExample"
-    name="Sizes"
-    description={`
-      There are 3 different pre-selected widths available for a \`Modal\`, as well as a last-resort
-      option to set a custom width. Click on each button to view a sample Modal of the specified size.
-      All Modals have a max width of 100%.
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        showCode={false}
+        type="do"
+        description="Use Modal when a response is required from the user. Clearly communicate what response is expected and make the action simple and straight forward, such as clicking a button to confirm. The most common responses will be related to confirming or canceling."
+        defaultCode={`
+          <iframe src="https://codesandbox.io/embed/trusting-wu-c8514?fontsize=14&hidenavigation=1&theme=light&view=preview"
```

Comment:
It looks like this is all copy/paste with a different src? We should DRY this out if possible — perhaps a function accepting a `src` and returning the template literal?

Reply:
Thought about that, but we may want different sizing each time. We could abstract out into something with 3 sizes eventually